index.d.ts

```diff
@@ -148,6 +150,27 @@ declare global {
     readonly bounds: Rect
   }
 
+  interface ParameterValues {
+    [key: string]: string
```

Comment:
Maybe change this to `[key: string]: any` since once we do the metadata support, that's what the type will be. This way we won't have to go through a round of breaking developer plugins

Reply:
Ah yeah was wondering about this. Was thinking another option might be to do `[key: string]: string | {suggestion: string, metadata: any}`, when we add that. Any thoughts on that?

Reply:
My thinking was that if someone does `addSuggestions([{ name: "foo", data: { id: "47:17" } }, ...])` and the user chooses the "foo" option, that we'd fire an event which contains `{ parameterName: { id: "47:17" }`. I.e. that the parameter gets set to the value specified in the `data` property of the suggestion directly.

I would imagine that in most cases the data would contain all the needed information for the plugin to do its job. Consider for example a "find node" plugin. You'd stick the name of the node in the `name`, and the id of the node in the `data`. Once the plugin runs, there is no need for the plugin to receive the data. The node id is all that is needed.

If a plugin does in fact need the displayed string as well, they can easily duplicate that string nested somewhere in the `data` property.

So as a result, the parameter that we'd pass back to the plugin can be of any format and so we should just use `any`